### DISCUSSION TITLE
Request for discussion and demo to open source community groups

### DISCUSSION THREAD

**Discussion by mlieberman85** [crev-dev/cargo-crev#467] [General]
Hi, a few of us in the [OpenSSF](https://openssf.org/) and [CNCF](https://www.cncf.io/), particularly around the supply chain security space, are interested in learning more about crev and seeing if there are areas we can collaborate. Would anyone be interested in attending one or more of the community meetings to discuss crev, maybe show a demo or presentation?

**Comment by dpc** [comment]
Hi. Since it doesn't look like anyone more willing has appeared, maybe I can try after all. I'm just very time constrained, so if you don't mind not a lot of preparation, I can always talk it over, demo  how it works and maybe answer some questions.

**Reply by matthiasdebernardini** [reply]
I would love that personally! How can I help?

**Comment by laurentsimon** [comment]
On OSSF side, scorecard https://github.com/ossf/scorecard#connect-with-the-scorecards-community would be a good fit, then maybe the best practice WG https://github.com/ossf/wg-best-practices-os-developers#meeting-times

[mlieberman85](https://github.com/mlieberman85), how about on the CNCF side?

**Reply by mlieberman85** [reply]
FOr CNCF the Security Technical Advisory Group (TAG Security) is probably the best spot.
